frontend/src/views/MovieSessionDetailsScreen.vue

```diff
@@ -68,7 +68,7 @@ export default {
     async fetchMovieSession (id) {
       try {
         this.loading = true;
-        const { data: session } = await this.axios.get(`${import.meta.env.VITE_API_URL}/api/cinema/movie_sessions-${id}`, {
+        const { data: session } = await this.axios.get(`${import.meta.env.VITE_API_URL}/api/cinema/movie_sessions-${id}/`, {
```

Comment:
Are you sure this endpoint exists on backend?
Also don't see screenshot of responses from developer tools